www/front_src/src/Header/SwitchThemeMode/index.tsx

```diff
@@ -1,104 +1,74 @@
+/* eslint-disable hooks/sort */
```

Comment:
In order to respect the hooks sort, you can extract the useAtom to an higher component that will wrap `SwitchThemeMode` component